Dockerfile.jupyter

```diff
@@ -9,6 +9,9 @@ RUN conda install requests-kerberos -y
 
 USER $NB_USER
 
+RUN pip install --upgrade pip
+RUN pip install --upgrade --ignore-installed setuptools
```

Comment:
Is the `--ignore-installed` needed?

Reply:
yes. it was required.

Reply:
Why?

Reply:
without it I got this error:  

```Step 7/25 : RUN pip install --upgrade setuptools
 ---> Running in a103e27aaeca
Collecting setuptools
  Downloading setuptools-36.6.0-py2.py3-none-any.whl (481kB)
Installing collected packages: setuptools
  Found existing installation: setuptools 23.0.0
Cannot remove entries from nonexistent file /opt/conda/lib/python3.5/site-packages/easy-install.pth
The command '/bin/sh -c pip install --upgrade setuptools' returned a non-zero code: 1
```

I googled until I found the --ignore-installed could get around it.  See:   https://github.com/ContinuumIO/anaconda-issues/issues/542